scripts/config-diff.sh

```
@@ -0,0 +1,491 @@
+    print_subheader "Commands"
+    
+    # Files to exclude from comparison (repo-specific)
+    local excluded_files=("sync.md" "config-diff.md")
+    
+    # Get all command files from both directories
+    local repo_commands=()
+    local user_commands=()
+    
+    if [[ -d "$REPO_COMMANDS_DIR" ]]; then
+        while IFS= read -r -d '' file; do
+            local basename=$(basename "$file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)